solver/src/main/java/org/chocosolver/solver/constraints/ISetConstraintFactory.java

```diff
@@ -369,6 +371,19 @@ default Constraint setsIntsChanneling(SetVar[] sets, IntVar[] ints, int offset1,
         );
     }
 
+    /**
+     * Channeling constraint stating that int[i] = v + offset, where v is the ith element of the sorted elements of set.
+     * If <i>i</i> is out of bounds with set, int[i] = nullValue.
+     * @param set A set variable.
+     * @param ints An array of integer variables.
+     * @param nullValue An int.
+     * @param offset An int.
+     * @return A sortedSetIntsChanneling constraint.
+     */
+    default Constraint sortedSetIntsChanneling(SetVar set, IntVar[] ints, int nullValue, int offset) {
+        return new Constraint("test", new PropSortedIntChannel(set, ints, nullValue, offset));
```

Comment:
Oups, I forgot to put a proper constraint name when submitting the PR... Can I let you fix this ?